### ISSUE TITLE
16-5 堆叠双向LSTM中文分词标注 样本丢失？

### ISSUE THREAD

**Issue by ghost** [Qinbf/Deep-Learning-Tensorflow2#4]
您好，代码在 TF2.2 平台上运行，模型训练时发现样本丢失很多，造成准确率很低

Epoch 1/30
268/268 [==============================] - 35s 132ms/step - loss: 0.2812 - accuracy: 0.2256 - val_loss: 0.1402 - val_accuracy: 0.2960
Epoch 2/30
268/268 [==============================] - 30s 112ms/step - loss: 0.1266 - accuracy: 0.3040 - val_loss: 0.1166 - val_accuracy: 0.3045
Epoch 3/30
268/268 [==============================] - 30s 110ms/step - loss: 0.1075 - accuracy: 0.3112 - val_loss: 0.1023 - val_accuracy: 0.3102
Epoch 4/30
268/268 [==============================] - 30s 112ms/step - loss: 0.0952 - accuracy: 0.3159 - val_loss: 0.0922 - val_accuracy: 0.3139
Epoch 5/30
268/268 [==============================] - 30s 112ms/step - loss: 0.0851 - accuracy: 0.3196 - val_loss: 0.0840 - val_accuracy: 0.3171
